code/game/machinery/shuttle/shuttle_engine_types.dm

```diff
@@ -299,11 +299,11 @@
 	for(var/reagent in fuel_reagents)
 		reagent_amount_holder += fuel_reagents[reagent]
 
-/obj/machinery/power/shuttle/engine/liquid/burn_engine(percentage = 100)
+/obj/machinery/power/shuttle/engine/liquid/burn_engine(percentage = 100, seconds_per_tick)
 	. = ..()
-	var/true_percentage = 1
+	var/true_percentage = seconds_per_tick * percentage / 100
 	for(var/reagent in fuel_reagents)
-		true_percentage *= reagents.remove_reagent(reagent, fuel_reagents[reagent]) / fuel_reagents[reagent]
+		true_percentage *= reagents.remove_reagent(reagent, fuel_reagents[reagent] * true_percentage) / (fuel_reagents[reagent] * true_percentage)
```

Comment:
This does nothing; you're multiplying the existing expression by 1.

Reply:
To the existing expression, yes, that's intended to cancel out. The point here is to multiply how many units of fuel `remove_reagent` removes by `true_percentage` so that fuel consumption isn't much higher or lower than it should be depending on physics tickrate and the ship's throttle setting.

Reply:
Ah, yeah I see that now. Apologies!